txpipe/plotting/correlations.py

```diff
@@ -264,6 +267,117 @@ def make_plot(corr, obs_data, theory_data, fig=None, xlogscale=True):
     plt.subplots_adjust(wspace=0.05, hspace=0.05)
     return plt.gcf()
 
+
+def make_plot_ratios(corr, obs_data, theory_data, fig=None, xlogscale=True):
+    import matplotlib.pyplot as plt
+    nbin_source = obs_data[0]['nbin_source']
+    nbin_lens = obs_data[0]['nbin_lens']
+
+    ny = nbin_source if types[corr][1] == 'source' else nbin_lens
+    nx = nbin_source if types[corr][2] == 'source' else nbin_lens
+
+
+    if corr == XIP:
+        name = r"\xi_+(\theta) \mathrm{ ratios}"
+        ymin = 5e-7
+        ymax = 9e-5
+        auto_only = False
+        half_only = True
+    elif corr == XIM:
+        name = r"\xi_-(\theta) \mathrm{ ratios}"
+        ymin = 5e-7
+        ymax = 9e-5
+        auto_only = False
+        half_only = True
+    elif corr == GAMMA:
+        ymin = 5e-7
+        ymax = 2e-2
+        name = r'\gamma_T(\theta) \mathrm{ ratios}'
+        auto_only = False
+        half_only = False
+    elif corr == W:
+        ymin = 2e-4
+        ymax = 1e-0
+        name = r'w(\theta) \mathrm{ ratios}'
+        auto_only = True
+        half_only = False
+    elif corr == EE:
+        name = r"C_\ell^{EE} \mathrm{ ratios}"
+        ymin = 2e-12
+        ymax = 9e-8
+        auto_only = False
+        half_only = True
+    elif corr == ED:
+        ymin = 2e-10
+        ymax = 2e-6
+        name = r"C_\ell^{ED} \mathrm{ ratios}"
+        auto_only = False
+        half_only = False
+    elif corr == DD:
+        ymin = 2e-8
+        ymax = 1e-4
+        name = r"C_\ell^{DD} \mathrm{ ratios}"
+        auto_only = True
+        half_only = False
+    elif corr == GAMMAX:
+        ymin = 5e-7
+        ymax = 2e-2
+        name = r"\gamma_X(\theta) \mathrm{ ratios}"
+        auto_only = False
+        half_only = False
+
+    plt.rcParams['font.size'] = 14
+    f = fig if fig is not None else plt.figure(figsize=(nx*3.5, ny*3))
+    ax = {}
+    
+    axes = f.subplots(ny, nx, sharex='col', sharey='row', squeeze=True)
+    for i in range(ny):
+        if auto_only:
+            J = [i]
+        elif half_only:
+            J = range(i+1)
+        else:
+            J = range(nx)
+        for j in range(nx):
+            a = axes[i,j]
+            if j not in J:
+                f.delaxes(a)
+                continue
+
+            for index, obs in enumerate(obs_data):
+
+                res = obs[(corr, i, j)]
+                theta_th, xi_th = theory_data[index][(corr, i, j)]
+                if len(res) == 2:
+                    theta, xi = res
```

Comment:
looks like in this case (when there's no error bars) this still plots xi not xi / xi_th